pkg/proxy/iptables/proxier_test.go

```diff
@@ -4393,6 +4414,7 @@ func TestInternalTrafficPolicyE2E(t *testing.T) {
 	}{
 		{
 			name:                  "internalTrafficPolicy is cluster",
+			line:                  getLine(),
```

Comment:
ah, `t.Helper()` wouldn't help quite as much with this, although because all the tests use `t.Run(tc.name, ...` now you can just search for the name string from the failure message rather than the line number

Reply:
The name string is also not-great because Go helpfully (!!) converts " " to "_" so I can't cut-paste it.  Also, we have cases in different tests which use the same name.